dev-app-post/main/default/profiles/Admin.profile-meta.xml

```diff
@@ -335,6 +370,176 @@
         <field>Case.Verbal_Order__c</field>
         <readable>true</readable>
     </fieldPermissions>
+    <fieldPermissions>
+        <editable>true</editable>
+        <field>Case.EDRD_Additional_information__c</field>
```

Comment:
Additional information needs to be changed to Additional Information @rahulshyamrajsingh

Reply:
The field's label has been changed because the build is giving us issues when we attempt to modify the API name.